py3-platformdirs.yaml

```diff
@@ -32,13 +39,24 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: 5ec69d86917fb1ac23207893ccd598f8af805ba5
 
-  - name: Python Build
-    runs: |
-      python -m build
-      python3 -m installer -d "${{targets.destdir}}" dist/*.whl
-      find ${{targets.destdir}} -name "*.pyc" -exec rm -rf '{}' +
-
-  - uses: strip
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-packagename}}
+    description: ${{vars.pypi-package}} for python${{range.key}}
+    dependencies:
+      provider-priority: ${{range.value}}
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
+    test:
+      pipeline:
+        - uses: python/import
+          with:
+            python: python${{range.key}}
+            import: ${{vars.pypi-packagename}}
+        - uses: strip
```

Comment:
Opps need to remove this strip

Reply:
it doesnt hurt. and if this built any c-python bits you'd want it, right?

Reply:
> it doesnt hurt. and if this built any c-python bits you'd want it, right?

It doesn't hurt. But this strip is on a test block there's already a strip on the building pipeline.

Reply:
ah. funny. yes, fix.